lib/CustomFields/pages/ViewCustomFieldsSettings/ViewCustomFieldsSettings.js

```diff
@@ -63,9 +67,16 @@ const ViewCustomFieldsSettings = ({
   } = useSectionTitleFetch(okapi, backendModuleName.toUpperCase());
 
   const { calloutRef } = useLoadingErrorCallout(customFieldsFetchFailed || sectionTitleFetchFailed);
+  const paneTitleRef = useRef(null);
 
   const noDataSaved = isEmpty(customFields) && !sectionTitle.value;
 
+  useEffect(() => {
+    if (paneTitleRef.current) {
+      paneTitleRef.current.focus();
+    }
+  }, []);
```

Comment:
@BogdanDenis should `paneTitleRef.current` be passed here as a param since it's used inside `useEffect`?

Reply:
@mkuklis Yes! I've added it to dependencies list